pico/cart.c

```diff
@@ -868,6 +869,7 @@ int PicoCartInsert(unsigned char *rom, unsigned int romsize, const char *carthw_
       elprintf(EL_STATUS|EL_ANOMALY, "starting in unknown hw configuration: %x", PicoIn.AHW);
     case 0:
     case PAHW_SVP:  PicoMemSetup(); break;
+    case PAHW_MCD|PAHW_VGM:

```

Comment:
Is this really right? Shouldn't this be split in 2 cases with fallthrough?

Reply:
Well both PAHW_MCD and PAHW_VGM are set, similar to SMS variants that are masked out a bit above. I can mask out PAHW_VGM too I guess.